.gitignore

```diff
@@ -125,6 +125,7 @@ typechain
 subgraph.yaml
 
 .DS_Store
+.vscode
```

Comment:
This is duplicate of what is already listed at the end of the file.

Reply:
oh, good catch. Thanks